nature/styleguide/color.md

```diff
@@ -1,363 +1,13 @@
-# Color
+# Colour
 
-Our color system creates a user-friendly interface for researchers, while also contributing to the character of Nature Portfolio. We organise colors into 3 groups; accent, background and user interface.
+Colour plays an important role in contributing to the character and identity of the Springer Nature visual style. Our colour system applies a core palette globally, to create a cohesive user experience for all our brands. Brand specific colours are used as an accent, such as highlighting top stories and illustrations.
 
-* Accent colors derive from the brand color pallete. We use them sparingly, such as for the journal identity or infographics.
-* Background colors are used for supporting material, such as article cards and illustrations. 
-* UI colors are the primary pallete for designing interactive elements, and are compliant with Web Content Accessibility Guidelines AA standard contrast ratios. We’ve also developed a wider range of neutral greys for the hierarchy and density of information in our use cases. 
+## Accessbility
+The colour system is designed to be compliant with accessibility standards. Springer Nature is committed to ensuring our products and services are usable by the largest possible audience, including people with vision, hearing, cognitive or motor impairments. We support WCAG 2.1 level AA which requires 4.5:1 contrast for text.

```

Comment:
A couple of suggested changes here to make it a bit more concise and grammatically correct.

```suggestion
The colour system is designed to be accessible. Springer Nature is committed to ensuring our products and services are usable by the largest possible audience. This includes people with visual, hearing, cognitive or motor impairments. We support WCAG 2.1 level AA which requires 4.5:1 contrast for text.

```